### PR TITLE
Update index.tsx

### PR DESCRIPTION
**Pull request by aytajj**

### CODE REVIEW

**Review by Kieran-McIntyre — CHANGES_REQUESTED** [review]
Hi @aytajj!

Thanks for this. This change looks good, but since we're using TypeScript here, we need to update the interface [here](https://github.com/Kieran-McIntyre/react-native-section-alphabet-list/blob/master/src/components/ListLetterIndex/types.ts) with the new prop. That's why we're seeing the unit tests fail.

I would give updating this a go, and I would also add a unit test to the component to test that the custom style is applied to the `FlatList` element. Please let me know if you would like any help/guidance.

Thanks!

**Comment by aytajj** [comment]
@Kieran-McIntyre, thank you for reply, I changed pull request. Can you please check?

**Comment by Kieran-McIntyre** [comment]
> Details

Hi @aytajj. Thanks for making that change. The unit tests are still failing. Can you please fix them? Once they are updated, I will take another look. Thanks!

**Comment by aytajj** [comment]
Dear @Kieran-McIntyre, I think it will work now. Can you please check?

**Comment by Kieran-McIntyre** [comment]
> Dear @Kieran-McIntyre, I think it will work now. Can you please check?

Hi @aytajj. The tests are still failing. Can you please run the tests locally before pushing your commit to ensure they're passing? Run command `yarn test` in your terminal.

**Comment by aytajj** [comment]
Hi @Kieran-McIntyre, I am sorry,  letterListContainerStyle must be optional.  I added question mark. Test was successful. Thank you

**Comment by aytajj** [comment]
@Kieran-McIntyre can you please check? All tests was successful.

**Comment by Kieran-McIntyre** [comment]
> @Kieran-McIntyre can you please check? All tests was successful.

Thanks @aytajj. One last thing, please could you update the `README.md` with the new prop?

File: src/components/AlphabetList/types.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export interface AlphabetListProps extends Partial<SectionListProps<IData>> {
   indexLetterStyle?: TextStyle,
   indexLetterContainerStyle?: ViewStyle,
   indexContainerStyle?: ViewStyle,
+  letterListContainerStyle?: CSSProperties,
```
Comment (Kieran-McIntyre) [inline src/components/AlphabetList/types.ts:26]: This should also be `ViewStyle` here

```suggestion
  letterListContainerStyle?: ViewStyle,
```

File: src/components/ListLetterIndex/index.test.tsx [hunk 2]
```diff
@@ -153,6 +153,34 @@ describe('ListLetterIndex', () => {
         // Assert.
         expect(indexItemLabelElements[0].props.style).toEqual(expectedStyle)
     })
+    it('letterListContainerStyle > should apply custom style', () => {
```
Comment (Kieran-McIntyre) [inline src/components/ListLetterIndex/index.test.tsx:156]:
```suggestion

    it('letterListContainerStyle > should apply custom style', () => {
```

File: src/components/ListLetterIndex/index.test.tsx [hunk 3]
```diff
@@ -153,6 +153,34 @@ describe('ListLetterIndex', () => {
...
+        const { getAllByTestId } = render(
+            <ListLetterIndex
+                sectionData={sectionData}
+                onPressLetter={mockPressLetterFn}
+                letterListContainerStyle={letterListContainerStyle}
+            />
+        )
 
+        const flatList = getAllByTestId("flatList");
+        // Assert.
```
Comment (Kieran-McIntyre) [inline src/components/ListLetterIndex/index.test.tsx:182]:
```suggestion
      
          // Assert.
```

File: src/components/AlphabetList/types.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { SectionListData, SectionListProps, TextStyle, ViewStyle } from "react-native";
+import { SectionListData, SectionListProps, TextStyle, ViewStyle, CSSProperties } from "react-native";
```
Comment (Kieran-McIntyre) [inline src/components/AlphabetList/types.ts:1]:
```suggestion
import { SectionListData, SectionListProps, TextStyle, ViewStyle } from "react-native";
```

**Review by Kieran-McIntyre — COMMENTED** [review]
Looks good, just final minor comments 👍

**Comment by aytajj** [comment]
Hi @Kieran-McIntyre, I've updated the pull request, please check it out. Thanks.

**Comment by aytajj** [comment]
Hi, @Kieran-McIntyre, Would you please check?

**Review by Kieran-McIntyre — APPROVED** [review]

**Comment by Kieran-McIntyre** [comment]
> Hi, @Kieran-McIntyre, Would you please check?

@aytajj  this has now been merged and published. Your change can be found in `v2.1.0`. Thanks!

**Comment by aytajj** [comment]
@Kieran-McIntyre , Thank you!

**Pull request merged**